src/app/utils/field-validation/validators/dateValidator.ts

```diff
@@ -59,10 +57,8 @@ const pastOnlyValidator: DateValidator = (response) => {
  */
 const futureOnlyValidator: DateValidator = (response) => {
   // Today takes two possible values - a min (here) and max (in makePastOnlyValidator)
-  // Subtract 12 hours here to account for up to UTC - 12 timezone
-  // This allows validation to pass as long as user is on the correct date (locally)
-  // Even if they are in a different timezone
-  const todayMin = moment().utc().subtract(12, 'hours').startOf('day')
+  // Dates are converted to use local timezones when loaded by the DateField so no conversion required
+  const todayMin = moment()
```

Comment:
Same here for server side validation

Reply:
should be fixed in [47cbc25](https://github.com/opengovsg/FormSG/pull/6096/commits/47cbc25f83d6d8f198ee104e9ca1651ae5721452)